recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-kernel/kernel-module-gles/kernel-module-gles.bbappend

```diff
@@ -0,0 +1,14 @@
+PVRKM_URL = "git://github.com/xen-troops/pvr_km.git"
+BRANCH = "vgpu-dev"
+SRCREV = "${AUTOREV}"
+
```

Comment:
that was in the original file: will fix here, but there is yet another copy of this in Dom0